website/docs/community/resources/long-lived-discussions-guidelines.md

```diff
@@ -0,0 +1,52 @@
+
+- [Help](https://discourse.getdbt.com/c/help/19)
+  - This is a Q&A style forum where you can ask the dbt Community for help with specific questions about dbt, dbt Cloud, data modeling, or anything else you want a definitive answer on.
+  - This category is for questions which can plausibly have a _single correct answer_.
+    - ✅ How do I debug this Jinja error?
+    - ✅ How do I set up CI in dbt Cloud on GitHub?
+    - ❌ What is the best way to do marketing attribution? (More general Discussions like this are perfect for the [In-Depth Discussions](https://discourse.getdbt.com/c/discussions/21) category)
+- [Show and Tell](https://discourse.getdbt.com/c/show-and-tell/22)
+  - This is the place to show off all of the cool things you are doing in dbt. Whether it’s a new macro, design pattern, or package, post here to show the community what you are up to!
+- [In-Depth Discussions](https://discourse.getdbt.com/c/discussions/21)
+  - Share anything you’re thinking about that has to do with dbt or analytics engineering! This is a great place to jot down some thoughts to share with the community or spark a discussion on a topic that’s currently interesting you.
+
+## Inclusivity on the Community Forum
+
+We are **strongly** committed to building a community where everyone can feel welcome.
+
+The dbt community started with people who were not traditionally considered “technical”, did not have ownership over technical systems, and were often left out of organizational decision-making. We came together to learn from each other, solve hard problems together, and to help build a new discipline where data folks have greater ownership over our own work. It really matters to us that everyone feels like they can ask questions and engage, no matter their professional or personal background.
+
+Technical forums have the potential to replicate harmful power structures, and can feel intimidating or hostile. We are working hard to create and sustain an inclusive environment through community-building, technological solutions, inclusive content, and diverse contributors. This is a long-term project, and we will continue to iterate and make improvements.
+
+If you have any ideas or feedback on how to make this space friendlier or more inclusive please let us know, either on the community Slack in the #community-strategy channel or via email at [community@dbtlabs.com](mailto:community@dbtlabs.com). We want to hear from you!
+
+## Following new and ongoing Discussions
+
+The best way to stay up to date is to [browse the forum](https://discourse.getdbt.com/) directly. You can also Track or Watch specific threads or the whole category to receive updates on them without commenting.
+
+Each category also has a companion Slack channel (#advice-dbt-for-beginners, #show-and-tell and #in-depth-discussions). You can reply to the initial post in Slack and it will be added as a comment on the forum thread, allowing you to participate from inside Slack if you prefer.
```

Comment:
```suggestion
Each category also has a companion Slack channel ([#advice-dbt-for-beginners](https://getdbt.slack.com/archives/CBSQTAPLG), [#i-made-this](https://getdbt.slack.com/archives/C01NH3F2E05) and #in-depth-discussions). You can reply to the initial post in Slack and it will be added as a comment on the forum thread, allowing you to participate from inside Slack if you prefer.
```

Reply:
trying to embed slack links. i couldn't find the channel "#in-depth-discussions". was it renamed? 

#show-and-tell was renamed to "#i-made-this"